firefox-ios/Client/Frontend/TabContentsScripts/LoginsHelper.swift

```diff
@@ -111,6 +151,32 @@ class LoginsHelper: TabContentScript {
         }
     }
 
+    func sendMessageType(_ message: FieldFocusMessage) {
+        switch message.fieldType {
+        case .username:
+            logger.log("Parsed message username",
```

Comment:
@razvanlitianu

PART1: This is where the magic will happen, once JS side is approved we will use this to know when to show the accessory view. Which when tapped will show the bottom sheet

We need to make small mechanism for this but I can show you or you can look into how other areas of autofill like Addresses / Credit card works. 

See PART2 for how to inject passwords into the page